docs/DATA_FORMATS_INPUT.md

```diff
@@ -104,9 +104,23 @@ but can be overridden using the `name_override` config option.
 
 #### JSON Configuration:
 
-The JSON data format supports specifying "tag keys". If specified, keys
-will be searched for in the root-level of the JSON blob. If the key(s) exist,
-they will be applied as tags to the Telegraf metrics.
+The JSON data format supports specifying "tag_keys", "string_keys", and "json_query". 
+If specified, keys in "tag_keys" and "string_keys" will be searched for in the root-level 
+and any nested lists of the JSON blob. All int and float values are added to fields by default. 
+If the key(s) exist, they will be applied as tags or fields to the Telegraf metrics.  
+If "string_keys" is specified, the string will be added as a field.
+
+The "json_query" configuration is a gjson path to an JSON object or list of JSON objects.  
+If this path leads to an array of values or single data point an error will be thrown.  If this 
+configuration is specified, only the result of the query will be parsed and returned as metrics.  
+
+Object paths are specified using gjson path format, which is denoted by object keys 
+concatenated with "." to go deeper in nested JSON objects.  
+Additional information on gjson paths can be found here: https://github.com/tidwall/gjson#path-syntax
+
+The JSON data format also supports extracting time values through the config "json_time_key" and "json_time_format". 
+If "json_time_key" is set, "json_time_format" must be specified.  The "json_time_key" describes the name of the field containing time information.  The "json_time_format" must be a recognized Go time format.  
+More info on time formats can be found here: https://golang.org/src/time/format.go
```

Comment:
Use `https://golang.org/pkg/time/#Parse` as the link.  This other link is still not great but the audience of this documentation is not Go developers.

Try to wrap around 78 chars